runtime/sema/type.go

```diff
@@ -2153,9 +2153,9 @@ func (t *VariableSizedType) IsImportable(results map[*Member]bool) bool {
 	return t.Type.IsImportable(results)
 }
 
-func (*VariableSizedType) IsEquatable() bool {
+func (v *VariableSizedType) IsEquatable() bool {
 	// TODO:
-	return false
+	return v.Type.IsEquatable()
```

Comment:
Maybe also remove the TODOs here and in other places (e.g: in `(*ConstantSizedType) IsEquatable()`)